hass_nabucasa/ai_api.py

```diff
@@ -0,0 +1,32 @@
+"""This module provides voice API functionalities."""
```

Comment:
I think we can merge these 2 new classes, it should be for voice also, but as that is in use its more trickier